web/static/js/app.jsx

```diff
@@ -217,7 +213,7 @@ var SearchBox = React.createClass({
     }
   },
 
-  onChange: function(e) {
```

Comment:
I actually don't mind listing unused function arguments when they're meaningful like this.